build/Mstc/MSTC.Web/scripts/mstc/event-booking.js

```diff
@@ -134,7 +136,7 @@
 	var getEvents = function () {
 		resetProps();
 		$.ajax({
-			url: '/umbraco/api/event/BookableEvents?futureEventsOnly=true&withSlotsOnly=true',
+			url: `/umbraco/api/event/BookableEvents?futureEventsOnly=true&withSlotsOnly=true&limitBooking=${limitBooking}`,

```

Comment:
Interesting way to pass a javascript parameter, not thought of doing it like that before!

Reply:
👍